UIKit/_UICache.h

```diff
@@ -0,0 +1,17 @@
+#import <Foundation/NSObject.h>
+
+NS_CLASS_AVAILABLE_IOS(7_0) @interface _UICache : NSObject
+
+- (instancetype)init;
+
+- (void)setObject:(id)object forKey:(id <NSCopying>)key;
+- (id)objectForKey:(id)key;
+- (void)removeObjectForKey:(id)key;
+- (void)removeAllObjects;
+- (NSArray *)cacheKeys NS_AVAILABLE_IOS(8_2);
```

Comment:
Do we know what kind of data go into this array?

Reply:
I’m sorta stumped on it, I’d only tested the clearsCacheOnLowMemoryWarnings and clearsCacheOnApplicationBackground (which is always set to YES by default) I assume an example of this that is actively used is related to network cache (as that is always purged upon jumping to another app or the springboard)
Maybe file paths?

Reply:
There are several class declarations such as `_UIFontCacheKey` and `_UIImageCacheKey` spread throughout frameworks (notably UIFoundation and UIKit). Maybe it refers to those?

Reply:
@chrisharper22 I was actually looking at that as a possibility about 10 min ago, I don’t think so, only because the values that return calling them with FLEX seem related to images that are cached with a .car file (so when it refers to “cached” images, I believe it means images within the .car file and not actual device cache. Or if not, .cpbitmap also seems like it falls under the same logic, except those are built to be actual cached files of images (that applications don’t control, see `/var/mobile/Library/Caches/MappedImageCache` on a device for more info on what type of images are cached (wallpaper, clock hands, Battery Center stuff, etc.)

Reply:
@chrisharper22 something else that supports my theory is that all the image types described (Keyboard images, Status Bar items, etc.) are the types of images found when extracting the assets from  `/System/Library/PrivateFrameworks/UIKitCore.framework/Artwork.bundle/Assets.car`

Reply:
(@dlevi309 I'm pretty sure PoomSmart was asking about the data type, meaning is the type `NSArray<NSString *> *`, etc.)

Reply:
Explored those kind of objects on FLEX and found that this array isn't necessary of NSString. Omitting type is fine by me.